src/ophys_etl/modules/suite2p_registration/__main__.py

```diff
@@ -250,22 +253,55 @@ def run(self):
         # with current ophys processing pipeline. In the future this output
         # should be removed and a better data storage format used.
         # 01/25/2021 - NJM
-        motion_offset_df = pd.DataFrame({
-            "framenumber": list(range(ops.item()["nframes"])),
-            "x": delta_x,
-            "y": delta_y,
-            "x_pre_clip": ops.item()['xoff'],
-            "y_pre_clip": ops.item()['yoff'],
-            "correlation": ops.item()["corrXY"],
-            "is_valid": is_valid,
-        })
+        if suite2p_args['nonrigid']:
```

Comment:
Since there were already suggestions to use a better format for data storage, should we store this information in a new format? Especially when you're storing numpy arrays, which I'm guessing it's the shifts for each block.

Reply:
ROIs are stored in a similar text format. 

I was reading the benefit of HDF5 vs text. HDF5 files are much smaller and reads faster. But if this data less on the order of kilobytes or single digit megabytes, it might not matter and not be worth the time rewriting this to a different format.

Reply:
Going to say this is a larger discussion beyond this ticket. For reference for a 40k sized movie, the file is 48MB with the nonrigid shifts. We'll definitely have to update folks that if we change this as they likely have code that assumes the plain text file. For reference, using pandas to load the CSV it's still pretty instant.

Reply:
A separate question I have: this information is only used for QC purposes and not used by downstream steps of the pipeline right?

Reply:
That is correct.

Reply:
If it's used by downstream parts of the pipeline, there should be a schema to validate the name and type.

Reply:
Name and type of the file sure.

Reply:
https://github.com/AllenInstitute/ophys_etl_pipelines/blob/main/src/ophys_etl/schemas/_roi_schema.py

Here's an ROI schema used in trace extraction. It includes the name and type of all arguments used as the inputs of that pipeline.

Reply:
Okay, I will say that this is beyond the scope of this ticket and we should open another one to add this feature if we want it.

Reply:
Finished adding the column check in unittesting. @mikejhuang take a look and let me know if it satisfies.

Reply:
Looks good, thanks for the discussion/changes. It helps me to clarify the principles of the design choices so I'm not just emulating patterns.